### PR TITLE
bump versions to v2021-01-28

### PR DESCRIPTION
**Pull request by KtorZ**
# Issue Number

N/A

# Overview

- [x] I have bumped all versions to 2021-01-28

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: "3.5"
 
 services:
   cardano-node:
-    image: inputoutput/cardano-node:1.24.2
+    image: inputoutput/cardano-node:1.25.1
```
Comment (piotr-iohk) [inline docker-compose.yml:5]: Since cardano-node 1.25.1 is not yet officially released, there is no 1.25.1 tag on docker hub yet... :shrug:

**Review by piotr-iohk — APPROVED** [review]

**Review by Anviking — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (KtorZ) [reply docker-compose.yml:5]: Ah!

Comment (KtorZ) [reply docker-compose.yml:5]: We're living on the edge.

Comment (KtorZ) [reply docker-compose.yml:5]: https://github.com/input-output-hk/cardano-node/releases/tag/1.25.1

:thinking:  ?

Comment (piotr-iohk) [reply docker-compose.yml:5]: Yes it's tagged, but no docker image for 1.25.1 until it's released. 

https://hub.docker.com/r/inputoutput/cardano-node/tags?page=1&ordering=last_updated

**Comment by KtorZ** [comment]
bors retry

**Pull request merged**